client/blocks/product-purchase-features/product-purchase-features-list/index.jsx

```diff
@@ -243,7 +247,13 @@ export default connect(
 	{
 		recordBusinessOnboardingClick: partial(
 			recordTracksEvent,
-			'calypso_plan_features_onboarding_click'
+			'calypso_plan_features_onboarding_click',
+			{}
+		),
+		recordReturnToDashboardClick: partial(
+			recordTracksEvent,
+			'calypso_plan_features_returnToDashboard_click',
```

Comment:
This is an invalid event name, it will be rejected: PCYsg-4S2-p2

lower-case and underscores 😉